hrm-domain/hrm-core/case/caseService.ts

```diff
@@ -608,6 +608,11 @@ export const generalisedCasesSearch = async (
       ctx,
     );
 
+    // monitor the number of cases returned by hrmdb
+    console.log(
```

Comment:
We should start trying to use the level specific log calls rather than just `console.log` - they don't make much difference right now but will make life easier when we add the capability to filter log output by level. In this case I think `console.info` would be appropriate